src/EditorFeatures/Core.Wpf/InlineHints/InlineParameterNameHintsTagger.cs

```diff
@@ -4,14 +4,15 @@
 
 using System;
 using System.Collections.Generic;
+using Microsoft.CodeAnalysis.Editor.InlineHints;
```

Comment:
Adding namespace that's the same as the one in the file? Either unused or is there a typo somewhere?

Reply:
Fixed.